installing/installing_ibm_cloud_public/preparing-to-install-on-ibm-cloud.adoc

```diff
@@ -1,12 +1,12 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="preparing-to-install-on-ibm-cloud"]
-= Preparing to install on IBM Cloud VPC
+= Preparing to install on {ibm-cloud-title}
 include::_attributes/common-attributes.adoc[]
 :context: preparing-to-install-on-ibm-cloud
 
 toc::[]
 
-The installation workflows documented in this section are for IBM Cloud VPC infrastructure environments. IBM Cloud Classic is not supported at this time. For more information about the difference between Classic and VPC infrastructures, see the IBM link:https://cloud.ibm.com/docs/cloud-infrastructure?topic=cloud-infrastructure-compare-infrastructure[documentation].
+The installation workflows documented in this section are for {ibm-cloud-name} infrastructure environments. {ibm-cloud-name} classic is not supported at this time. For more information about the difference between classic and VPC infrastructures, see the IBM link:https://cloud.ibm.com/docs/cloud-infrastructure?topic=cloud-infrastructure-compare-infrastructure[documentation].
```

Comment:
@SNiemann15 , @alishaIBM , and @mjpytlak . I received confirmation from IBM that "IBM Cloud Classic" refers to IBM loud classic infrastructure, so "IBM CLoud Classic" is not a product name here. Also, because I changed "IBM Coud VPC" references to "IBM Cloud", I should remove the following sentences from the docs?

For more information about the difference between classic and VPC infrastructures, see the IBM link:https://cloud.ibm.com/docs/cloud-infrastructure?topic=cloud-infrastructure-compare-infrastructure[documentation].

An out-dated link at this stage of updating terms, IMO.

Reply:
Not sure. I think this is a question better suited for IBM product owners and I am not sure who that would be.